hugo/content/faqs/_index.md

```diff
@@ -2,87 +2,88 @@
+- Making your genomic data visualisations accessible to everyone, whether they have skills in bioinformatics or not.
 
-[[faq.section]]
-question = "Is it possible to add my genomic data to the Genome Portal while a manuscript is under review?"
-answer = "TBD"
+- Save yourself time by not having to install a genome browser or download large data files to your local computer.
 
-[[faq.section]]
-question = "How long does it take for my data to be displayed on the Genome Portal?"
-answer = "TBD"
+- Having the flexibility to decide what, when, and how you want your genomic data be displayed on the Genome Portal.
 
-## About the genome browser
+- Having a weblink to your data displayed on the Genome Portal, which can be included in the Data Accessibility Statement of your scientific publication.
 
-[[faq.section]]
-question = "What genome browser is used in the Genome Portal?"
-answer = "TBD"
+- Receiving assistance with sharing valuable genomic annotations in the SciLifeLab Data Repository that are rarely published.
 
-[[faq.section]]
-question = "How can I customise (e.g. change color, track label) the way how data tracks are displayed on the genome browser?"
-answer = "TBD"
+- Enhancing your team work by being able to easily share links to genomic regions of interest with your colleagues.
+{{< /faq_block >}}
 
-[[faq.section]]
-question = "What could be causing my data to display slowly?"
-answer = "TBD"
+{{< faq_block title="How can I get my data displayed on the Genome Portal?" >}}
+Please begin by checking that your data meets the minimal requirements listed on the <a href="/contribute" target="_blank">Contribute</a> page.
 
-## About citation guidelines
+If it does, feel free to reach out via email to <dsn-eb@scilifelab.se> or through the <a href="/contact" target="_blank">Contact</a> form. We would be happy to learn more about your genomic project, and will provide a brief form to collect additional details about your data.
+{{< /faq_block >}}
 
-[[faq.section]]
-question = "How do I cite the Genome Portal website?"
-answer = "TBD"
+{{< faq_block title="What data file formats are supported for display on the Genome Portal?" >}}
+The Genome Portal uses the JBrowse 2 genome browser to display genomic datasets. <a href="https://jbrowse.org/jb2/features/#supported-data-formats" target="_blank">JBrowse supports several formats</a> that are commonly used in genomics (e.g., BED, VCF, FASTA, GFF, among others), which could therefore be displayed in the Genome Portal. However, at the moment, we do not accept complete BAM files derived from shotgun sequencing, as they can be quite large and may impact performance. Users can, however, add and visualise BAM files as local data tracks in the Genome Portal’s genome browser.
+{{< /faq_block >}}
 
-[[faq.section]]
-question = "How do I cite a Species page from the Genome Portal?"
-answer = "TBD"
+{{< faq_block title="How can I add a data track to the genome browser of an existing Species page?" >}}
```

Comment:
Suggestion to clarify that this is not about submitting a new data track, but to add it to the JBrowse 2 instance in the users webpages.

```suggestion
{{< faq_block title="How can I display a local data track in the genome browser of an existing Species page?" >}}